ios/CodePush/CodePush.m

```diff
@@ -247,6 +247,20 @@ - (instancetype)init
  */
 - (void)initializeUpdateAfterRestart
 {
+    dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Should we move this out to another method? Somehing like "clearDebugUpdates" or whatever? It might help keep this core method slim.